### ISSUE TITLE
Inefficient `aria-current` attribute set when the argument is `false`

### ISSUE THREAD

**Issue by Jessidhia** [facebook/react#14619]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

Rendering an HTML element with `aria-disabled={false}` produces an HTML element with `aria-disabled="false"`, but that is a _truthy_ `aria-disabled` value.

https://codesandbox.io/s/71jq2zoyoj

**What is the expected behavior?**

When `aria-disabled={false}` is given, the attribute should be either omitted or set to the empty string.

See the specification at https://www.w3.org/TR/wai-aria-1.1/#aria-current. The wording is confusing, but `true` and `false` in the explanation for `aria-current` are not valid tokens -- that is just the evaluated state for the accessibility engine.

---

EDIT: Oops. reading _even further_ down the spec, it does seem that `"false"` is supposed to be a falsy value too, so this is not a "bug" as in wrong behavior per se. It'd still be better to omit the attribute entirely, though.

**Comment by nhunzaker** [comment]
Hey! I'm a bit confused about the relationship between `aria-current` and `aria-disabled`. Is this a bug with `aria-current`, `aria-disabled`, or when they are used together?

> It'd still be better to omit the attribute entirely, though.

Just for my understanding. Why? Is it because it assigns an attribute unnecessarily?
